server/utils/checkAuth.js

```diff
@@ -0,0 +1,17 @@
+const { customError } = require('../controllers/errors');
+const verifyToken = require('./jwt/verifyToken');
```

Comment:
export from utils index,
And remove this file to `middleware.js` file inside controller